app/controllers/evms_controller.rb

```diff
@@ -60,6 +60,15 @@ def index
     end
     @no_data = issues.blank?
     @no_data_incomplete_issues = @incomplete_issues.blank?
+    # export
+    respond_to do |format|
+      format.html
+      format.csv {
+        send_data(@project_evm.to_csv,
+                  :type => 'text/csv; header=present',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.